moped-editor/src/views/projects/projectsListView/ProjectsListViewMap.js

```diff
@@ -1,18 +1,62 @@
 import React, { useEffect } from "react";
 import { useQuery } from "@apollo/client";
+import { useLocation } from "react-router-dom";
+import { NavLink as RouterLink } from "react-router-dom";
+import MapDrawer from "./components/MapDrawer";
 import ProjectsMap from "./components/ProjectsMap";
 import Alert from "@mui/material/Alert";
+import Link from "@mui/material/Link";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemText from "@mui/material/ListItemText";
+import Paper from "@mui/material/Paper";
+import makeStyles from "@mui/styles/makeStyles";
+import { useProjectGeographies } from "./useProjectGeographies/useProjectGeographies";
 import { GET_PROJECTS_GEOGRAPHIES } from "src/queries/project";
-import { styleMapping } from "../projectView/ProjectStatusBadge";
+
+const useStyles = makeStyles(() => ({
+  content: {
+    display: "flex",
+    height: "100%",
+    width: "100%",
+  },
+}));
 
 const ProjectsListViewMap = ({
   mapQuery,
   fetchPolicy,
   setIsMapDataLoading,
+  searchWhereString,
+  advancedSearchWhereString,
 }) => {
+  const classes = useStyles();
+
+  /* Get search params to pass in project links for back button in Project Summary view */
+  const location = useLocation();
+  const queryString = location.search;
+
   /* Store map instance to call Mapbox GL methods where needed */
   const mapRef = React.useRef();
 
+  /* Store featured project IDs for sidebar items */
+  const [featuredProjectIds, setFeaturedProjectIds] = React.useState([]);
```

Comment:
whats a featured project ID? does this mean a selected project?

Reply:
yeahhhh i struggled with this! i also like `selectedProjectIds` since that is closest to what is actually happening. i'll change and review the code again to make sure i'm not flip-flopping between these words too. 🙏

Reply:
done ✅ thanks again!